agenta-web/src/components/AppSelector/AppSelector.tsx

```diff
@@ -287,7 +287,11 @@ const AppSelector: React.FC = () => {
                                     <Card
                                         className={classes.createCard}
                                         onClick={() => {
-                                            if (isDemo() && apps.length > 2) {
+                                            if (
+                                                isDemo() &&
+                                                selectedOrg?.is_paying == false &&
```

Comment:
it would be nice to have this as a function isPaying() which would call itself useProfileData or the context